Newsletters: add 335 (2025-01-03)

harding opened the pull request:
~~Opening as draft as I haven't had time to spell check or self edit.  Please don't waste your time reviewing until this is out of draft.~~

- [x] News items, spellcheck, self-edit @harding 
- [ ] Lede, releases/RCs, topic entries @harding 
- [x] Notable merges @Gustavojfe

harding commented:
Sorry I didn't have time to get back to this today.  It'll be about another 8 hours before I'm able to make updates.

jirijakes commented on _posts/en/newsletters/2025-01-03-newsletter.md:
I am not sure whether this question makes sense in English, but is the “changing” in the title meant as adjective or as verb?

Or in other words, which is closer to the intent:

- section about how we are changing Bitcoin consensus
- section about Bitcoin consensus that is in the middle of a change

If it makes no sense, please just ignore :-)

harding replied:
Talked to ChatGPT about this and I think it's the figure of speech known as _ellipsis_.  A full construction would be something like: "News about ideas for changing consensus rules", but I elide the parts that are likely to be understood: "News about ideas for" ('cause this is a newsletter) and "rules" (following longstanding convention of calling "consensus rules" just "consensus" for convenience).

It feels natural to me as a native speaker and seems in line with other stuff, e.g. the "Scaling Bitcoin" conference comes to mind (probably short for "Conference about ideas for scaling Bitcoin transactional throughput").  However, if you think it'll be too confusing to non-native speakers, let me know and we can use a different section title going forward.

jirijakes replied:
Thanks. The full construct “News about ideas for changing consensus rules” answers my question.

It is not too confusing. Even though I can interpret the title in two different ways, the difference is subtle and I don't think it makes sense to spend time on clarifying it.

However, I wanted to better understand before I decide how to translate the title. In our language I cannot elide that much from the full construct.

vostrnad replied:
The translation doesn't have to be exactly literal, for example "Změny konsenzu" would work just fine. (We can discuss this in the translation PR later.)

montana64 commented:
Meta Platforms (META) is experiencing a positive movement in its stock price, primarily driven by a recent price target increase from JMP Securities. The company is showing strong growth prospects despite facing some challenges. Here are the key factors influencing its movement:

1. **Core Advertising Business**: Meta's core advertising segment continues to demonstrate robust margins, contributing positively to its financial performance.

2. **Reality Labs Challenges**: While the Reality Labs division is facing losses, the overall valuation of Meta suggests significant upside potential, with projections of 28% annual returns over the next three years.

3. **Analyst Recommendations**: Despite concerns such as rising AI costs and a potential slowdown in revenue growth, analysts recommend holding onto Meta due to its market dominance and strategic positioning for future gains.

4. **Financial Performance**: Meta reported a 19% year-over-year revenue increase, reaching $40.6 billion in Q3 2024, and a 25% growth in EBITDA, indicating strong financial health.

5. **User Base Growth**: The company's global daily users have reached 3.29 billion, with a 4.8% year-over-year growth, and a 12% increase in Family Average Revenue Per Person, reflecting successful monetization strategies.

However, there are some concerns about potential stagnation in Facebook's active user base and competition from platforms like TikTok. Additionally, significant investments in Reality Labs and uncertainties surrounding AI adoption pose challenges to Meta's financial health in the near term.

— Generated by Alpha from Public.com

jirijakes commented on _posts/en/newsletters/2025-01-03-newsletter.md:
I think both of these two suggestions would work:

```suggestion
    inputs on the stack, which it can use as a parameter for
    `OP_TEMPLATEHASH` (e.g. to require an equal output value).
```

jirijakes replied:
```suggestion
    inputs on the stack, which can be used as a parameter for
    `OP_TEMPLATEHASH` (e.g. to require an equal output value).
```

vostrnad commented on _posts/en/newsletters/2025-01-03-newsletter.md:
```suggestion
- [LND v0.18.4-beta][] is a minor release of
```

vostrnad commented on _posts/en/newsletters/2025-01-03-newsletter.md:
```suggestion
- [Bitcoin Core 28.1rc2][] is a release candidate for a maintenance
```

vostrnad commented on en/internal/sources.md:
```diff
@@ -7,7 +7,8 @@ redirect_from: /internal/sources
 # Sources for the Optech Newsletter
 
-The _News_ section of the newsletter is currently sourced from the
+The _news_ and _changing consensus_ sections of the newsletter are
```
Shouldn't section titles be capitalized?

harding commented on _posts/en/newsletters/2025-01-03-newsletter.md:
> We insert the title "developer" out of necessity to prevent confusion between 0xB10C's nym and conflicting phrasing (e.g. a hex string).  -- https://github.com/bitcoinops/bitcoinops.github.io/pull/1942#discussion_r1796470529

I think "moonsettler" is an unusual string to appear here and could leave the reader wondering if they're missing something or we have a typo, so I add a title to provide context.  We don't usually provide titles for strings that are obviously names.

jirijakes replied:
I did not recall this conversation. Makes sense.

Note to myself: make sure my first mention in Optech News is under pseudonym “born in 1830” so that the section starts with “developer born in 1830 posted to…” :smiling_imp:

harding replied:
lol

jirijakes commented on _posts/en/newsletters/2025-01-03-newsletter.md:
```suggestion
  a steady state during an upgrade and then receives a `commitment_signed`
  message that requires the next commitment point to be available.
```

jirijakes commented on _posts/en/newsletters/2025-01-03-newsletter.md:
```suggestion
  pathfinding by improving the [probability density function (PDF)][probability
```

harding commented:
Made edits, or left replies, for all feedback (thanks @jirijakes and @vostrnad !).  Added lede and updated releases/RCs but didn't add topic entries (I'll have to do that another day).  Review section by @Gustavojfe -- especially great job this week!

Thanks everyone, and sorry I was behind schedule for this whole cycle.

jirijakes commented on _posts/en/newsletters/2025-01-03-newsletter.md:
```diff
@@ -0,0 +1,309 @@
+lang: en
+---
+
+This week's links to information about longstanding deanonymization
```
```suggestion
This week's newsletter links to information about longstanding deanonymization
```

bitschmidty commented on _posts/en/newsletters/2025-01-03-newsletter.md:
```suggestion
  Authentication Code (HMAC) and a nonce and allow a recipient to authenticate
```

shesek commented on _posts/en/newsletters/2025-01-03-newsletter.md:
Not quite. While this does allow spending multiple vaulted outputs together, it is missing two of `OP_VAULT` properties:

- No partial re-vaulting support, the vaulted outputs must be spent in full to the staging script.
- No dynamic construction of the destination tap leaf, which enables choosing the withdrawal destination address at trigger time. Vaults using moonsettler's proposal would be limited to using static pre-determined destination addresses, similarly to simple `OP_CTV`-only vaults.
